### DISCUSSION TITLE
Doks powered multi-page website for landing, blogs and guides.

### DISCUSSION THREAD

**Discussion by mrdotkg** [thuliteio/doks#122] [Showcase]
# Doks can be customised infinitely. Thank you @h-enk for open-sourcing it.

Website: [www.butterops.dev](https://www.butterops.dev)
Repo: [butterops/butterops.dev](https://github.com/butterops/butterops.dev)

Mobile             |  Desktop
:-------------------------:|:-------------------------:
<img src="https://user-images.githubusercontent.com/5073326/107184108-f3a52800-6a05-11eb-8cf6-3fa5601e6dbc.png" data-canonical-src="https://user-images.githubusercontent.com/5073326/107184108-f3a52800-6a05-11eb-8cf6-3fa5601e6dbc.png"  width="100%"  />  |  <img src="https://user-images.githubusercontent.com/5073326/107183452-dd4a9c80-6a04-11eb-9c77-ac2ccc221dd7.png" data-canonical-src="https://user-images.githubusercontent.com/5073326/107183452-dd4a9c80-6a04-11eb-9c77-ac2ccc221dd7.png" width="90%" />

**Comment by h-enk** [comment]
Hey @mrdotkg, thanks and thank you for using Doks: nice website and interesting content!

Are you OK with me adding [ButterOps Dev](https://www.butterops.dev/) to the Doks showcase (I guess so, but I have to ask)?

**Reply by mrdotkg** [reply]
Already glad :) that you liked the website and content. Feel free to showcase it.

Thank you.

**Comment by h-enk** [comment]
A little delayed, but here we are: https://getdoks.org/showcase/butterops/

**Reply by mrdotkg** [reply]
Thank you @h-enk.  Any idea why  https://gridsome.org/ looks just like a doks themed website?

**Reply by h-enk** [reply]
Shout-out to @tomtev for the brilliant design!

**Reply by epogrebnyak** [reply]
> 
> 
> Shout-out to @tomtev for the brilliant design!

Yeah, but I mean... it is just doks theme without any reference to it... maybe aske them to credit Doks theme? Even the Prologue in folder structure is there as in Doks. Shouldn't they have mentioned _"based on a Doks theme by @h-enk "_?

**Reply by h-enk** [reply]
No, it is the other way around. I based the UX + design of Doks on Gridsome. I translated it to Bootstrap and changed a few things — trying to keep in line with the design idea of Tommy Vedvik. See also: [Meet Doks: modern documentation theme for Hugo](https://henkverlinde.com/meet-doks-modern-documentation-theme-for-hugo/#user-experience)

I added a notice to the [MIT License](https://github.com/h-enk/doks/blob/master/LICENSE) of course, and Tommy is OK with using his work like so for Doks:

> Its perfectly fine :) Site looks great!
